### PR TITLE
[Entrypoints] Use LlumnixClient class in entrypoints

### PR DESCRIPTION
**Pull request by s5u13b**
Previously, we call `manager_generate`, `manager_abort` functions to replace with vLLM AsyncLLMEngine's apis `generate` and `abort` in entrypoints. In this PR, we support LlumnixClientVLLM which have the same `generate` and `abort` apis as vLLM AsyncLLMEngine, and therefore it's more user-friendly and maintainable.

### CODE REVIEW

File: llumnix/entrypoints/vllm/arg_utils.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+def add_cli_args(parser):
+    parser.set_namespace("llumnix")
+    parser = LlumnixEntrypointsArgs.add_cli_args(parser)
+    parser = EngineManagerArgs.add_cli_args(parser)
+    parser.set_namespace("vllm")
+    parser = AsyncEngineArgs.add_cli_args(parser)
+    cli_args = parser.parse_args()
+    return cli_args
+
+def get_args(cfg, parser, cli_args):
```
[the remark this answers is not in the document]
Comment (s5u13b) [reply llumnix/entrypoints/vllm/arg_utils.py:19]: I think get_args is more accurate. We check args implicitly. For user, this function is used to get args.

File: llumnix/entrypoints/vllm/api_server.py [hunk 2]
```diff
@@ -44,16 +41,18 @@
...
+llumnix_client: LlumnixClientVLLM = None
 
 
 # pylint: disable=unused-argument
 @asynccontextmanager
 async def lifespan(fastapi_app: FastAPI):
-    asyncio.create_task(llumnix_context.request_output_queue.run_server_loop())
-    asyncio.create_task(_background_process_outputs(llumnix_context))
+    print(llumnix_client)
+    print(llumnix_client.request_output_queue)
```
[the remark this answers is not in the document]
Comment (s5u13b) [reply llumnix/entrypoints/vllm/api_server.py]: Leave it for standalone pr.

File: llumnix/entrypoints/setup.py [hunk 3]
```diff
@@ -240,41 +244,38 @@ def setup_llumnix(engine_manager_args, engine_args, cfg):
...
+                                                            server_info,
+                                                            log_requests,
+                                                            log_request_timestamps)
 
-    return context
+    return llumnix_entrypoints_context
 
 # TODO(s5u13b): Fix the potential output token out-of-order issue caused by the migration.
-async def _background_process_outputs(llumnix_context):
+async def background_process_request_outputs(llumnix_client):
```
[the remark this answers is not in the document]
Comment (s5u13b) [reply llumnix/entrypoints/setup.py]: Done.

File: llumnix/entrypoints/setup.py [hunk 4]
```diff
@@ -39,17 +40,20 @@
 
 
 class LlumnixEntrypointsContext:
```
[the remark this answers is not in the document]
Comment (s5u13b) [reply llumnix/entrypoints/setup.py:42]: LlumnixClientVLLM is for vllm, but setup.py is used by any llm backends.

**Review by KuilongCui — APPROVED** [review]

**Pull request merged**